pyproject.toml

```diff
@@ -1,3 +1,12 @@
+[tool.djlint]
+blank_line_after_tag="load,extends,import,from"
+blank_line_before_tag="load,extends,block"
+max_line_length=120
+# _base.html excluded because of the bad </script> nesting after format
+exclude=".templates/layouts/_base.html"
+# both error codes excluded because of how we pass the variables into strings
+ignore="H008,T027"
```

Comment:
Add some description of what they are.

Reply:
Had a look at these ignores and they can be fixed easily. Fixing the quoting fixes both. Ignore should only be used for extreme cases or if it takes a lot of effort to fix. These seem very simple to fix, was there some other reason they were ignored?

Reply:
I fixed this using smart quote and that fixed all the issues, since we want to use smart quotes in those two examples anyway? No more ignore codes, no false positives.

Reply:
Make format re-formats it. Which is also a problem, if lint passes then re-format shouldn't be changing it.

Reply:
So the only possibility is to add it to ignore list?

Reply:
The false positive yes, but you need to investigate why re-formats changes the files when lint says it passed. That is either a config issue or a bug.

Reply:
Can't see it in existing open issues, the only thing I could find is `Formatter is a beta tool` so possibly they are not aligned and that's why the discrepancies. I will raise an issue.

Reply:
Once raised, it can be resolved. For reference the issue with base template is due to: https://github.com/djlint/djLint/issues/824

Reply:
They seem to use this format where they separate linter and formatter issues but I will try anyway. Yeah the `</script>` problem people already complain about, cool that the jinja formatting issue is also raised.

Reply:
Djlint is flaky as hell but at least some start in formatting html in runner.